### DISCUSSION TITLE
Custom run npx package with verdaccio registry

### DISCUSSION THREAD

**Discussion by nhatchimai111** [verdaccio/verdaccio#2137] [Q&A] [answered]
Hi all,
I have published my cli package on verdaccio local. I would like using npx command to run my cli package.
Example: `npx mycli --registry http://localhost:4873`
However the terminal show error
```
npm ERR! code E404
npm ERR! 404 Not Found - GET http://localhost:4873/mycli - no such package available
```
Can you help me.

**Comment by juanpicado** [comment, marked as answer]
As far I know `--registry` does not work work with `npx` https://github.com/zkat/npx/issues/133#issuecomment-386538367
